src/common/core/coretree.js

```diff
@@ -689,6 +690,9 @@ define([
 
             node = normalize(node);
 
+            //currently there is no reason to call the persist on a non-root object
+            node = getRoot(node);
+
```

Comment:
yeah, normalize works updwards
so though we agreed that a persist should always go on the complete containment, I though it would be nice to still start the normalization from the input node